resources/js/Components/MismatchRow.vue

```diff
@@ -193,18 +198,22 @@ function showDialog(e: Event) {
 <style lang="scss">
 @import "@wikimedia/codex-design-tokens/theme-wikimedia-ui";
 
-  a {
-    break-line-link {
-      width: 100%
-    }
-  }
+a {
+break-line-link {
+    width: 100%
+}
+}
```

Comment:
Indentation doesn't look good here

Reply:
maybe this came from a rebase or something, it is weird and incorrect :grimacing: